platform/android/export/export.cpp

```diff
@@ -2365,6 +2374,16 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 		return have_plugins_changed || first_build;
 	}
 
+	Error save_apk_expansion_file(const Ref<EditorExportPreset> &p_preset, const String &p_path) {
+		int version_code = p_preset->get("version/code");
+		String package_name = p_preset->get("package/unique_name");
+		String apk_file_name = "main." + itos(version_code) + "." + get_package_name(package_name) + ".obb";
+		String fullpath = p_path.get_base_dir().plus_file(apk_file_name);
```

Comment:
Let's wrap this logic into a `get_apk_expansion_full_path` method and use it both here and in `get_command_line_flags` below to get the `fullpath` variable.